scripts/crypt-gpg-pinentry

```diff
@@ -5,20 +5,25 @@
 
 // Check if we're running directly from git repo or if we're running
 // from a PEAR packaged version.
-$phpDir = '@php-dir@';
-if ($phpDir[0] === '@') {
+$path = '@php-dir@';
+if ($path[0] === '@') {
     // Git repo needs to set the include path as the pinentry program is not
     // invoked in the current directory.
-    set_include_path(
-        dirname(__FILE__) . DIRECTORY_SEPARATOR . '..'
-        . PATH_SEPARATOR . get_include_path()
-    );
+    $path = dirname(__FILE__) . DIRECTORY_SEPARATOR . '..';
 } else {
     // PEAR packaged version needs to set include path in case it is running
     // in a local PEAR tree that's not in the system PHP include path.
-    set_include_path($phpDir);
 }
 
+// Workaround for composer installs (#19914)
+$cdir = $path . DIRECTORY_SEPARATOR . '../Console_CommandLine';
+if (@is_dir($cdir)) {
```

Comment:
I'd like to avoid the error-suppression operator. I don't think it raises a warning if the dir does not exist.